tests/_support/Page/Framework.php

```diff
@@ -1129,4 +1129,18 @@ public function iUploadTheItemTypeCASEFile()
         return $this;
     }
 
+    /**
+     * @When /^I download the framework PDF file$/
+     */
+    public function iDownloadTheFrameworkPdfFile(): Framework
+    {
+        $I = $this->I;
+
+        $I->click('Export', '#itemInfo');
+        $I->waitForElementVisible('#exportModal a.btn-export-pdf');
+        $I->click('#pdfgen');
```

Comment:
This makes sure that the button is there to click, but there should be a check that a file is downloaded (and that it is a PDF file, possibly also that it contains something expected).